problem_builder/public/js/problem_builder.js

```diff
@@ -0,0 +1,40 @@
+window.ProblemBuilder = {
+
+    transformClarifications: function(element) {
+        element = $(element);
```

Comment:
Nit: I'd prefer `var $element = $(element);` for clarity. Actually, on second look, I think you should actually remove this line. The call to `transformExisting` will work fine with a DOM node rather than a jQuery object, and the only other use of `element` here actually needs a DOM node not a jQuery element.